src/ApiException.php

```diff
@@ -114,6 +114,39 @@ public static function createFromApiResponse(
         );
     }
 
+    /**
+     * @param string $basicMessage
+     * @param int $rpcCode
+     * @param array|null $metadata
+     * @param \Exception $previous
+     * @return ApiException
+     */
+    public static function createFromApiResponseREST(
```

Comment:
Nit, but we use Camelcase even for acronyms, and also, it'll be more readable if we put `Rest` before API:
```suggestion
    public static function createFromRestApiResponse(
```